File: object/account.py
class Account:
    def __init__(self, val):
        self.__category = val['분류']
        self.__ammount = val['금액']
        self.__date = val['결제일']
        self.__content = val['내용']
    
    @property
    def category(self):
        return self.__category
    
    @property
    def ammount(self):
        return self.__ammount
    
    @property
    def date(self):
        return self.__date

    @property
    def content(self):
        return self.__content

File: flutter_application_1/lib/object/account.dart
class Account {
  String category;
  int ammount;
  String date;
  String content;
  Account(
      {required this.category,
      required this.ammount,
      required this.date,
      required this.content});

  Map<String, dynamic> toMap() {
    return {
      'category': category,
      'ammount': ammount,
      'date': date,
      'content': content
    };
  }

  factory Account.fromJson(Map<String, dynamic> json) {
    return Account(
        category: json['properties']['분류']['select']['name'],
        ammount: json['properties']['금액']['number'],
        date: json['properties']['결제일']['date']['start'],
        content: json['properties']['내용']['title'][0]['text']['content']);
  }
}
